drivers/hmis/app/models/hmis/hud/custom_service.rb

```diff
@@ -38,6 +38,12 @@ def within_range?(range)
     date_provided.between?(range.begin, range.end)
   end
 
+  def display_name
+    return service_name if service_type.custom_service_category.name == service_name
+
+    "#{service_type.custom_service_category.name} - #{service_name}"
+  end
```

Comment:
I think we should probably use `service_type.name` instead of `service_name` because we don't keep that field up to date.